src/@types/interfaces.ts

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+export interface LanguagesInput {
```

Comment:
You are not using these type in the code (import them in *.tsx and *.ts files).
It sohould be input type of function defined in jsonToCol.tsx (jsonToCol.ts). 

Please change the extension of the file jsonToCol.tsx to jsonToCol.ts.